Cooper/ClientApp/src.v2/app/layouts/home/home.layout.html

```diff
@@ -1,10 +1,11 @@
-<div>
-  <h2>
-    Home page is not implemented yet.
-  </h2>
-  <nav>
-    <div (click)="goToProfilePage()">
-      <a class="pointed">Go to my page</a>      
-    </div>
-  </nav>
-</div>
\ No newline at end of file
+
+<div class="container">
+
+  <img class="image" src='assets/imageKeeper/house-repair.png'>
+  
+  <div class="text-block">
+    <p class="text"> Home page hasn't been implemented yet. </p>
+
+    <p><a class="pointed" (click)="goToProfilePage()">Go to my page</a></p>
```

Comment:
image, if everybody will create new class for each link

also, this is not imperative styling. if you want to use declarating style it should be name pointer-cursor or something like this